Tasks/DotNetCoreInstallerV0/task.json

```diff
@@ -68,6 +94,8 @@
         "WarningVersionNotFound": "Could not fetch download information for version %s. Please check if the version specified is correct. You can refer the link for supported versions => https://github.com/dotnet/core/blob/master/release-notes/releases.json. Falling back to creating convention based URL.",
         "CouldNotDownload": "Could not download installation package from this URL: %s Error: %s",
         "FailedToDownloadPackage": "Failed to download package for installation",
-        "PrependGlobalToolPath": "Creating global tool path and pre-prending to PATH."
+        "PrependGlobalToolPath": "Creating global tool path and pre-prending to PATH.",
+        "VersionMissingException": "If you don't use global.json then please defined a version.",
```

Comment:
define

Reply:
done

Reply:
I think we have resolve this. If not reopen this conversion.